README.md

```diff
@@ -143,9 +145,9 @@ This repo follows the Neural Query Engine taxonomy proposed in the paper (Figure
 </details>
 
 <details>
-  <summary>Discrete + Continuous (Entities + string/numerical Literals) (0)</summary>
+  <summary>Discrete + Continuous (Entities + numerical value) (1)</summary>
```

Comment:
I would rather not change the original classification in favor of one particular work. The big vision is still to be able to run queries against both strings and numerical literals

Reply:
And if we add NRN here, then LitCQD should go into this category first :)

Reply:
I did not pay enough attention to the categories. I totally agree with you! Thank you for your reminder!

Reply:
Is there anything that I can help with?

Reply:
Should I add a new pull request? I am not very familiar with the pull request system

Reply:
Oh, I am reading the github doc, and I think I am able to tackle it